### ISSUE TITLE
Erik's token can't be used everywhere

### ISSUE THREAD

**Issue by ErikBoesen** [ErikBoesen/Yalebot#83]
We should somehow use the token of the adder. This is risky security wise. Might need a privacy policy. :P

**Comment by dominiccoletti** [comment]
Can we encrypt it in the database?

**Comment by ErikBoesen** [comment]
No, because it needs to be directly passed to GroupMe rather than just match a hash of an inputted password. God, I wish GORT was open source.
